mypy_stubs/arango/cursor.pyi

```diff
@@ -1,3 +1,21 @@
-from typing import Any, Generator
+from typing import Any
 
-Cursor = Generator[Any, None, None]
+# Below is autogenerated with stubgen
```

Comment:
Neat, what is `stubgen`?

Reply:
It's a tool included with mypy that makes it easy to generate boilerplate stubs https://mypy.readthedocs.io/en/stable/stubgen.html#automatic-stub-generation-stubgen